analysis/src/PhysObject.cpp

```diff
@@ -47,7 +47,12 @@ dzErr(-9999),
 vx(-9999),
 vy(-9999),
 vz(-9999),
-
+////////////////////date:5/02/2023
```

Comment:
I really don't think we need to add comments with edit dates - this is what we have git for

Reply:
I used it for to track the changes for myself before PR. I have removed it now.